packages/cli/main.js

```diff
@@ -4,19 +4,40 @@ import getopts from "getopts";
 import hrp from "http-request-plus";
 import { format, parse } from "json-rpc-protocol";
 import { mapValues, omit } from "lodash";
-const WebSocket = require("ws");
+import * as WebSocket from "ws";
 
 async function main() {
-  const opts = getopts(process.argv.slice(2));
-  const method = opts._[0];
-  const params = mapValues(omit(opts, "_"), String);
+  async function call(method, params) {
```

Comment:
This function is not needed. We only request the server when the watch mode is `undefined`.

Reply:
As a solution, declare a constant dedicated to the got arguments, call the `getopts` with `stopEarly` as an option then check if the `watch` exist.
If it exists, subscribe to the server to get notifications.
If not, call the `getopts` without `stopEarly` and omit, cast the got params then request the server.

Reply:
Always run `getopts` with `stopEarly` first to parse global options then call it again without `stopEarly` to parse method parameters.